### PR TITLE
Contextualize the frontpage for paying/door access

### LINKED ISSUE #41: Contextual content on front page
If possible, when logged in, the content on the front page https://p2k16.bitraf.no/#!/ should vary wether you are a paying member or not. 

• Paying members should not see the "Become a paying Bitraf member" text.
• Non paying members should not see the door opening buttons.

### PR DESCRIPTION
**Pull request by omega**
If you are a paying member, dont show the "how to become paying member"
If you are a paying member, but not the door circle, show information on
how to get door access
If you are paying and have door access, show the door buttons.

Texts are kinda just from the issue and improvised.

Fixes #41

### CODE REVIEW

File: run-p2k16-web [hunk 1]
```diff
@@ -30,4 +30,5 @@ export STRIPE_PUBLIC_KEY='pk_test_hUoh3MEu4Oa8RjVFwZrFge4b'
 exec env/bin/gunicorn \
     --config "$basedir/infrastructure/gunicorn-config.py" \
     --reload \
+    --bind "0.0.0.0:5000" \
```
Comment (mastensg) [inline run-p2k16-web]: this seems unrelated to the rest of this commit, and it seems to me that it would make the production p2k16 accessible without going through the normal proxy

**Comment by jonnor** [comment]
Did you guys merge get this into master somehow? Can it be closed? @jenschr

**Comment by omega** [comment]
This was merged via another #50, so closing this now
